### PR TITLE
Chart

### PR DESCRIPTION
**Pull request by priolette**
- Added Statistics page to project using Chart.js
- Currently, only sums of all properties from all data entries are shown

### CODE REVIEW

File: Energy Saver/Services/ChartService.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            Chart chart = new Chart();
+            List<List<Taxes>> tableData = Serialization.ReadFromFile();
+            chart.Type = Enums.ChartType.Line;
+
+            Data data = new Data();
+            data.Labels = new List<string>() { "Gas", "Electricity", "Water", "Heating" };
+
+            LineDataset dataset = new LineDataset()
+            {
+                Label = "Amount payed all time",
```
Comment (Leminode) [inline Energy Saver/Services/ChartService.cs]: Should be "paid" instead of "payed". Could replace sentence with "Total paid amount"

**Review by Leminode — CHANGES_REQUESTED** [review]
Preliminary table looks good, just change the "payed" part

File: Energy Saver/Services/ChartService.cs (hunk 1 above)
Comment (priolette) [reply Energy Saver/Services/ChartService.cs]: Fixed the typo

**Review by Leminode — APPROVED** [review]

File: Energy Saver/Services/ChartService.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
+﻿using ChartJSCore.Helpers;
+using ChartJSCore.Models;
+using Energy_Saver.Model;
+
+namespace Energy_Saver.Services
+{
+    public class ChartService : IChartService
+    {
+        public Chart CreateChart()
```
Comment (Tauras-Rim) [inline Energy Saver/Services/ChartService.cs:9]: Chart data should be changed. Its very difficult to understand. Maybe change x axis to time(years and months) and y axis to total amount of all taxes at a given month (electricity + water + heating + gas)?

**Review by Tauras-Rim — APPROVED** [review]

**Pull request merged**